docs/plugins/woodpecker-plugins/plugins.json

```
@@ -35,6 +35,11 @@
       "docs": "https://codeberg.org/qwerty287/woodpecker-regex-check/raw/branch/main/docs.md",
       "verified": false
     },
+    {
+      "name": "Gitea comment",
```

```suggestion
      "name": "Gitea Pull comment",
```
-> https://github.com/markopolo123/gitea-comment-plugin/blob/main/docs.md should also be changed there ...

of course better names are welcome ;) ... and the name also could stay if it would be posible to overload the index as setting so we can comment on issues ...

---

@markopolo123 would depend on what you wana go for 🚀